pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceEphemeralData.java

```diff
@@ -63,9 +81,17 @@ public void setDisabled(boolean flag) {
         this.disabled = flag;
     }
 
+    @NotNull
+    public Map<String, AdvertisedListener> getAdvertisedListeners() {
+        if (this.advertisedListeners == null) {
+            return Collections.unmodifiableMap(Collections.EMPTY_MAP);
```

Comment:
nit: I think `EMPTY_MAP` is already unmodifiable.

Reply:
I have fix the code